composer/loggers/logger_destination.py

```diff
@@ -137,3 +137,22 @@ def log_symlink_artifact(
         """
         del state, log_level, existing_artifact_name, symlink_artifact_name, overwrite  # unused
         pass
+
+    def get_file_artifact(
+        self,
+        artifact_name: str,
```

Comment:
Would a logger destination need access to the `state` to get the file? I presume not, in which case this signature is fine, but just wanted to raise the question.

Reply:
Hm... I don't think so. State should be divorced of the files accessible. I modeled signature after `get_file`